src/code/templates/fv-genome-challenge.js

```diff
@@ -67,15 +68,35 @@ class YourDrakeView extends React.Component {
 class TargetDrakeView extends React.Component {
 
   static propTypes = {
+    showIntro: PropTypes.bool,
     org: PropTypes.object
   }
 
+  state = {
+    introComplete: false
+  }
+
+  handleIntroComplete = () => {
+    this.setState({ introComplete: true });
+  }
+
   render() {
-    const { org } = this.props;
+    const { showIntro, org } = this.props,
+          { introComplete } = this.state,
+          completeDisplayStyle = { display: showIntro && !introComplete ? 'none' : 'flex' },
+          introAnimation = <AnimatedSprite
+                              classNames='target-gizmo-animation'
+                              frames={5} frameWidth={915} duration={1600}
+                              onEnd={this.handleIntroComplete} />,
+          // instantiating with {display:none} allows images to preload during animation
+          completeView = <div className='target-drake-gizmo' style={completeDisplayStyle}>
+                            <div id="target-drake-label">Target Drake</div>
+                            <OrganismView id="target-drake" org={org} width={300} />
+                          </div>;
     return (
-      <div className='target-drake-gizmo'>
-        <div id="target-drake-label">Target Drake</div>
-        <OrganismView id="target-drake" org={org} width={300} />
+      <div className='target-drake-view'>
+        { showIntro && !introComplete ? introAnimation : null }
```

Comment:
It may not currently ever be false, but I was thinking we might want to only show it the first time a the layout was visited, for instance, in which providing some control over its appearance would make sense.